container-search/src/main/resources/configdefinitions/container.search.schema-info.def

```diff
@@ -5,15 +5,15 @@ namespace=search.config
 schema[].name string
 
 ## The id of the summary class
-# schema[].summaryclass[].id int
+schema[].summaryclass[].id int
```

Comment:
the corresponding documentdb[].summaryclass[].id is never used, so you can remove this id.